client/src/routes/App/FileTransfer/FileTransfer.js

```diff
@@ -292,17 +292,17 @@ class FileTransfer extends PureComponent {
           isSelectorEnabled: false,
         });
       },
-      onDone: (file, meta) => {
-        if (file !== undefined) {
-          if (Array.isArray(file)) {
-            file.forEach(file => {
-              file.getBlob((err, blob) => download(blob, file.name));
-            });
+      onDone: (files) => {
+        if (files !== undefined) {
+          if (Array.isArray(files)) {
+            multiDownload(
+              // make regular File from webtorrent File https://github.com/webtorrent/webtorrent/blob/v1.9.7/lib/file.js#L13
+              files.map(file => new File([file.getBlob()], file.name, {type: file.type}))
+            );
           }
           else {
-            download(file, meta.name, meta.type);
+            download(files);
           }
-        }
         this.resetState();
       },
```

Comment:
There seems to be a syntax error here. A closing bracket is missing

Reply:
fixed! don't know how that sneaked in 🤔